ccm_web/client/src/components/Home.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+
+import FeatureCard, { FeatureCardProps } from './FeatureCard'
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined'
+import LibraryBooksOutlinedIcon from '@material-ui/icons/LibraryBooksOutlined'
+import MergeTypeIcon from '@material-ui/icons/MergeType'
+import PersonAddIcon from '@material-ui/icons/PersonAdd'
+import PersonAddOutlinedIcon from '@material-ui/icons/PersonAddOutlined'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    minWidth: 275
+  }
+}))
+
+function Home (): JSX.Element {
```

Comment:
I'm wondering if we should make this component a `Page` or a container, rather than a simple component under `components`. In my limited experience, distinguishing between common, elemental components and higher-order components (that use other components and contain data fetching) seems to be a useful way to organize a React project.

Reply:
If we go the page route, we can either create a `Layout` component later or use some framing/header/footer/navigation stuff in each separate page.

Reply:
@ssciolla So you want a containers folder like myla at the same level as components?

Reply:
Yeah, at that level. Do you have strong preference between `containers` or `pages`? I have a slight preference for the `page` idea, I think it's a little less abstract than `container`.

Reply:
Pages makes more sense to me. I don't know how a component is really all that different than a 'container' anyways